bin/parse_dmc_classification.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+import sys
+
+if " " in sys.argv[1] :
+  seqName = sys.argv[1].split(" ")[0]
+else :
+  seqName = sys.argv[1]
```

Comment:
```suggestion
seqName = sys.argv[1].split(" ", 1)[0]
```
Should hopefully capture the first instance before the space if it exists.

Reply:
Definitively better and easier yes, thanks !